src/Services/Air/Air.js

```diff
@@ -13,9 +13,16 @@ module.exports = (settings) => {
   const log = (settings.options && settings.options.logFunction) || console.log;
   return {
     shop(options) {
+      if (typeof options.async !== 'undefined' && options.async === true) {
```

Comment:
Check for undefined is not needed as we compare to true with strong type matching

Reply:
Changed as commented.